### PR TITLE
Updates to personal keymaps and improvements to Prime_O and DZ60 layout templates

### PR DESCRIPTION
**Pull request by SpacebarRacecar**
- Updated personal userspace and keymaps
- Added iso split spacebar layout template to dz60
- Added personal prime_o keymap
- Fixed prime_o layout template so that the position of the keys makes sense and is not random anymore, also adjusted default keymap accordingly

### CODE REVIEW

**Comment by SpacebarRacecar** [comment]
I just realized that a dz60 iso split spacebar layout got pushed just yesterday. Anyway I have also added an ascii representation which is still missing :)

**Comment by noroadsleft** [comment]
You've actually changed the matrix on the DZ60/LAYOUT_60_iso_split.

All changes to `dz60.h` and `info.json` need to be reverted back to QMK's current master.

**Comment by SpacebarRacecar** [comment]
Yeah, I just noticed that aswell and changed it again. If you still want me to revert back to master I can do that.

**Comment by noroadsleft** [comment]
You fixed it, it's good now.

Here's another suggestion, but it doesn't pertain to your submitted files.

I noticed that you're filing this PR from your fork's `master` branch. While this isn't a major issue now, later pull requests are likely to have merge conflicts because the commit history will have diverged between your master and QMK's.

It is highly recommended for QMK development - regardless of what is being done or where - to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.

I have the QMK Firmware repository set up as a remote repo in Git, so I can check it directly and compare it to both my fork, and my local files.

I do the following every time I need to make a branch (using Git via MSYS on Windows 8) :

```
git fetch upstream
git checkout master
git pull upstream master
git push origin master
```

This retrieves any recent changes to QMK's master - nicknamed `upstream` - and syncs them to my fork, which is nicknamed `origin`. Then when I start development on something, I create a new branch for whatever I'm working on with `git checkout -b dev_branch`, which creates a branch `dev_branch` that is based on the current branch, and then checks out the new branch. I'll make all my changes to `dev_branch`, then when they're ready, I'll do a pull request from `noroadsleft:dev_branch` to `qmk:master`.

If you need any help with this just ask.

**Comment by SpacebarRacecar** [comment]
Thank you, I'm pretty inexperienced when it comes to git and your comment helps a lot! I will do that from now on :)

**Comment by noroadsleft** [comment]
This is basically for you:

https://docs.qmk.fm/#/newbs_best_practices

**Comment by SpacebarRacecar** [comment]
I'm closing this pull request and will reopen another pull request following noroadsleft's suggestions.

File: keyboards/primekb/prime_o/prime_o.h [hunk 1]
```diff
@@ -19,22 +19,22 @@
 #include "quantum.h"
 
 #define LAYOUT( \
-	K000, K001, K002, K003, K004, K005, K006, K007, K010, K011, K012, K013, K014, K015, K016, K017, \
-	K020, K021, K022, K023, K024, K025, K026, K027, K030, K031, K032, K033, K034, K035, K036, K037, \
-	K040, K041, K042, K043, K044, K045, K046, K047, K050, K051, K052, K053, K054, K055, K056, K057, \
-	K060, K061, K062, K063, K064, K065, K066, K067, K070, K071, K072, K073, K074, K075, K076, K077, \
-	K080, K081, K082, K083, K084, K085, K086, K087, K090, K091, K092, K093, K094, K095, K096, K097  \
+	K001, K008, K013, K018, K023, K029, K035, K041, K046, K052, K058, K064, K071, K076, K081, K086, \
```
Comment (drashna) [inline keyboards/primekb/prime_o/prime_o.h]: Why the change to the matrix here?

Comment (SpacebarRacecar) [reply keyboards/primekb/prime_o/prime_o.h]: I did it so that the numbers match up with the actual numbers written on the pcb.
